### PR TITLE
Add technical stuff

### PR DESCRIPTION
**Pull request by mattbk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,7 @@ Teams](https://gratipay.com/) for examples of both accepted and rejected applica
 
 **Want to help review applications?** Read the [docs](https://inside.gratipay.com/howto/review-teams), 
 and then [dive in](https://github.com/gratipay/applications/issues)!
+
+**Approving or Rejecting Teams** Once a decision has been made in the application issue, Gratipay contributors with admin access can modify the team status attribute on the team page.
+
+![Approve a team](https://github.com/gratipay/team-review/blob/mattbk-patch-2/team-approve.png)
```
Comment (rohitpaulk) [inline README.md]: We'll need to link this to `master` instead of `mattbk-patch-2` to ensure that the image lasts :)

Comment (mattbk) [inline README.md]: Yeah, I know. If patch-2 is merged first, then I can change patch-1 to point to the right place.

Comment (rohitpaulk) [inline README.md]: Done :)

**Comment by chadwhitacre** [comment]
I think we should keep the README very simple, and we should put specifics about how to review on the Inside Gratipay doc.

**Pull request closed without merging**